tests/stdlib/tmath.nim

```diff
@@ -1,4 +1,15 @@
-import math, random
+discard """
+  action: run
+  output: '''[Suite] random int
```

Comment:
What would you recommend then to do runtime tests? Using only `block`s & `assert`s becomes hard to maintain with a large number of test cases. Going for less runtime behavior tests in general is also not beneficial, and as far as I can see `unittest` is the best tool for that job, isn't it?